server/routes/temporaryabsences/checkTemporaryAbsenceController.test.ts

```diff
@@ -3,13 +3,17 @@ import request from 'supertest'
 import cheerio from 'cheerio'
 import { appWithAllRoutes } from '../__testutils/appSetup'
 import TemporaryAbsencesService from '../../services/temporaryAbsencesService'
+import raiseAnalyticsEvent from '../../raiseAnalyticsEvent'
 
 import Role from '../../authentication/role'
 import config from '../../config'
 
 jest.mock('../../services/temporaryAbsencesService')
 const temporaryAbsencesService = new TemporaryAbsencesService(null, null) as jest.Mocked<TemporaryAbsencesService>
 let app: Express
+const flash = jest.fn()
+
+jest.mock('../../raiseAnalyticsEvent')
```

Comment:
this direct import of raiseAnalyticsEvent doesn't really fit the rest of the way the app is structured - I think this was copied in from DPS core.

We'd usually inject in our dependencies, it would also be nice to be able to switch it off - don't really want it to be trying to fire in integration tests for instance. 

Maybe we should add a ticket to change this to a more conventional dependency - which doesn't do anything when disabled.

Reply:
Raised DCS-1408 for this